client/src/frontend/containers/JournalDetail/index.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import PropTypes from "prop-types";
+import CheckFrontendMode from "global/containers/CheckFrontendMode";
+import EntityCollection from "frontend/components/composed/EntityCollection";
+import lh from "helpers/linkHandler";
+import HeadContent from "global/components/HeadContent";
+import EventTracker, { EVENTS } from "global/components/EventTracker";
+import {
+  useSelectSettings,
+  useSelectJournal,
+  usePaginationState,
+  useFilterState,
+  useUrlFromState
+} from "hooks";
+import { RegisterBreadcrumbs } from "global/components/atomic/Breadcrumbs";
+import Layout from "frontend/components/layout";
+import { pageChangeHandlerCreator } from "helpers/pageChangeHandlerCreator";
+
+export default function JournalDetailContainer({ location, match }) {
+  const { journal, journalResponse } = useSelectJournal(match);
+  const settings = useSelectSettings();
+  const { paginationState, handlePageChange } = usePaginationState(
+    location,
+    20
+  );
+  const { filterState, updateFilterState } = useFilterState(location, {
+    collectionOrder: match.params.id
+  });
+  // TODO: add useDispatchIssues hook
+  // TODO: add useSelectIssues hook
```

Comment:
It felt like diminishing returns to keep mocking hooks when the api is a couple days away. The re-fetch on filter state change logic also needs to be added here.